parsing_scripts/load_initial_data.py

```diff
@@ -0,0 +1,122 @@
+
+        new_port.variants = port.get('variants')
+        new_port.description = port.get('description')
+        new_port.homepage = port.get('homepage')
+        new_port.epoch = port.get('epoch')
+        new_port.platforms = port.get('platforms')
+        new_port.long_description = port.get('long_description')
+        new_port.revision = port.get('revision', 0)
+        new_port.closedmaintainer = port.get('closedmaintainer')
+        new_port.license = port.get('license')
+        new_port.installs_libs = port.get('installs_lib')
+        new_port.replaced_by = port.get('replaced_by')
+        new_port.save()
+
+        try:
+            new_port.categories.add(*port['categories'])
+        except KeyError:
+            pass
+
+        try:
+            for maintainer in port['maintainers']:
+                try:
+                    name = maintainer['email']['name'].lower()
+                    domain = maintainer['email']['domain'].lower()
+                    email_provided = True
+                except KeyError:
+                    name = None
+                    domain = None
+                    email_provided = False
+
+                try:
+                    github = maintainer['github'].lower()
+                    github_provided = True
+                except KeyError:
+                    github = None
+                    github_provided = False
+
+                # Check if the maintainer already exists in Database:
+                if email_provided is True and github_provided is True:
+                    try:
+                        db_maintainer = Maintainer.objects.get(name=name, domain=domain, github=github)
+                        db_maintainer.ports.add(new_port)
+                        continue
+                    except ObjectDoesNotExist:
+                        pass
+                elif email_provided is False and github_provided is True:
+                    try:
+                        db_maintainer = Maintainer.objects.get(name__isnull=True, github=github)
+                        db_maintainer.ports.add(new_port)
+                        continue
+                    except ObjectDoesNotExist:
+                        pass
+                elif email_provided is True and github_provided is False:
+                    try:
+                        db_maintainer = Maintainer.objects.get(name=name, domain=domain, github__isnull=True)
+                        db_maintainer.ports.add(new_port)
+                        continue
+                    except ObjectDoesNotExist:
+                        pass
+
+                # If the maintainer was not found in the database already, then only this code will run:
+                new_maintainer = Maintainer()
+                new_maintainer.name = name
+                new_maintainer.domain = domain
+                new_maintainer.github = github
+                new_maintainer.save()
+                new_maintainer.ports.add(new_port)
```

Comment:
Wouldn't the `get_or_create()` method work here?
https://docs.djangoproject.com/en/2.2/ref/models/querysets/#get-or-create